generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/temporal/AfterOrAt.feature

```diff
@@ -6,18 +6,17 @@ Feature: User can specify that a datetime date is more than, or the same as, a s
     And foo is of type "datetime"
     And foo is anything but null
 
-  @ignore #141 add user configurable datetime granularity
   Scenario: User requires to create a datetime field with date (YYYY-MM-DD) values within a given month that are after or at a specified date
     Given foo is after or at 2018-01-01T00:00:00.000Z
     And the generator can generate at most 6 rows
```

Comment:
this should be either updated to use the datetime granularity.

or removed, if it's covered in other tests

Reply:
we have already a good coverage for  datetime granularity here: https://github.com/ScottLogic/datahelix/blob/master/generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/GranularTo-DateTime.feature 
But this test tests the default value of datetime granularity. The desciption is wong, will fix it.

Reply:
should we just delete this test instead

Reply:
Maybe.. I noticed that there is some repetition in the test suite.  We can definitely delete some tests without reducing the coverage. But as the execution is very fast, so I am fine with keeping them. 
I can work on that in another PR though, if we think there is a good added value.